### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by yjkimjunior**

### CODE REVIEW

File: CONTRIBUTING.MD [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+[Styleguides](#styleguides)
+  * [Git Commit Messages](#git-commit-messages)
+  * [JavaScript Styleguide](#javascript-styleguide)
+  * [Specs Styleguide](#specs-styleguide)
+  * [Documentation Styleguide](#documentation-styleguide)
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the [Kleros Code of Conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to [kleros@github.com](mailto:kleros@github.com).
```
Comment (clesaege) [inline CONTRIBUTING.MD]: Is this email set?

Comment (clesaege) [reply CONTRIBUTING.MD]: I tried but it's invalid. Let's remove the conduct report.

Comment (yjkimjunior) [reply CONTRIBUTING.MD]: Oops, yeah my bad.

**Comment by clesaege** [comment]
Thanks, I merge.

**Pull request merged**